### DISCUSSION TITLE
Menu items for App Secrets documentation

### DISCUSSION THREAD

**Discussion by justo-mend** [renovatebot/renovate#30696] [Suggest an Idea]
### Tell us more.

There is a new page for App Secrets in the Docs. But there is no item for it in the menu structure.
Also for the new page on Migrating App Secrets. Both are relevant only for the Cloud service.

Question: Should we create menu items for either of these pages?
If so, what is the best place for these items to appear?

Thanks for your input.

**Comment by rarkins** [comment]
Can you include screenshots to help explain?

**Comment by justo-mend** [comment]
This is the [App Secrets page](https://docs.renovatebot.com/getting-started/app-secrets/). (Note: it is for the cloud hosted apps only)

<img width="429" alt="image" src="https://github.com/user-attachments/assets/9a019112-4b6a-4663-afe3-4ae06e2c8d96">

**Comment by justo-mend** [comment]
This is the [Migrating App Secrets](https://docs.renovatebot.com/getting-started/migrating-secrets/) page. (Note: Also for the cloud-hosted apps only)

<img width="430" alt="image" src="https://github.com/user-attachments/assets/e4f29da1-f627-44b7-9d26-c6c085004ad0">

**Comment by rarkins** [comment]
We might want to have a new menu subsection specifically for docs which apply only to the Mend apps like this

**Reply by justo-mend** [reply]
This is what I was thinking, too.

**Comment by justo-mend** [comment]
Suggestions for placement of the new menu items for Mend's cloud apps.

Option 1: A top level menu item
![image](https://github.com/user-attachments/assets/1dd2d8cb-60ad-4096-8b76-191e29a41b33)

Option 2: Menu item contained under the "Configuration" menu
![image](https://github.com/user-attachments/assets/415869c9-944e-4bf2-a226-97ab75756935)

Even though both of these items are technically configuration (and could fit under the Configuration node), I prefer option 1, because it allows room for future pages that relate to the Cloud apps that are not configuration.

**Reply by HonkingGoose** [reply]
I like option 1 best, as top-level means it's always visible. 🙂

**Reply by rarkins** [reply]
I prefer option 1 because the contents will likely be more than just configuration

**Reply by justo-mend** [reply]
Thanks for providing your opinion.
Stand by for a PR with updates to the menu - using Option 1 - the top-level item.

**Reply by HonkingGoose** [reply]
See issue:

- #30711
